backend/src/org/commcare/session/CommCareSession.java

```diff
@@ -356,14 +358,14 @@ public Suite getCurrentSuite() {
         return null;
     }
 
-    public void stepBack() {
```

Comment:
Probably a separate PR/idea but all this evalContext passing around the CommCareSession feels a little silly, is there any reason not to just pair the evalContext with the Session at this point? (as seems to be the case in the SessionWrappers)? Does the evalContext ever "diverge" from the CommCareSession such that commCareSession having a getter would return the wrong thing?